conda_smithy/linter/lints.py

```diff
@@ -225,14 +225,24 @@ def lint_should_be_empty_line(meta_fname, lints):
 
 
 def lint_license_family_should_be_valid(
-    about_section, license, needed_families, lints
-):
-    license_family = about_section.get("license_family", license).lower()
-    license_file = about_section.get("license_file", None)
-    if not license_file and any(
-        f for f in needed_families if f in license_family
-    ):
-        lints.append("license_file entry is missing, but is required.")
+    about_section,
+    license,
+    needed_families,
+    lints: List[str],
+    is_rattler_build: bool = False,
+) -> None:
+    lint_msg = "license_file entry is missing, but is required."
+    if not is_rattler_build:
+        license_family = about_section.get("license_family", license).lower()
+        license_file = about_section.get("license_file", None)
+        if not license_file and any(
+            f for f in needed_families if f in license_family
+        ):
+            lints.append(lint_msg)
+    else:
+        license_file = about_section.get("license_file", None)
```

Comment:
I think technically the second parameter is not necessary and you could move this to the outer scope since the code is teh same in both branches.

Reply:
change a little the structure - lmk